widgetssdk/src/main/java/com/glia/widgets/call/CallController.java

```diff
@@ -231,7 +231,9 @@ public void onHoldChanged(boolean isOnHold) {
     public void engagementEnded() {
         Logger.d(TAG, "engagementEndedByOperator");
         stop();
-        dialogController.dismissDialogs();
+        if (!isOngoingEngagementUseCase.invoke()) {
+            dialogController.dismissDialogs();
+        }
```

Comment:
I'm a little confused about git diff. I was expecting to see "TODO re-enable during MOB-2523" here, added by [this pull request](https://github.com/salemove/android-sdk-widgets/pull/703). Is there a chance that you checkout some outdated `development` state?

Reply:
That is interesting that these don't show here, I actually deleted the TODO-s during development

Reply:
Git shows some conflicts for this branch. Maybe they are related to this question.